src/pages/thank-you.tsx

```diff
@@ -116,7 +126,7 @@ export default ({ data: { site } }) => {
 
   return (
     <Layout site={site} noFooter>
-      <SEO title="Thank You | Joe Previte" />
```

Comment:
`title` wasn't used at all `SEO`

Instead on:
1. Post page: `title` is used from `frontmatter` when you pass it as `mdx?.frontmatter`
2. Other pages: `config.siteTitle` where you have "Joe Previte - Developer and Teacher"

Reply:
Ahh got it! That makes sense